code/modules/mob/living/damage_procs.dm

```diff
@@ -109,5 +109,9 @@
 	if(drowsy)		apply_effect(drowsy, DROWSY, blocked)
 	if(agony)		apply_effect(agony, AGONY, blocked)
 	if(flammable)	adjust_fire_stacks(flammable)
-	if(ignite)		IgniteMob()
+	if(ignite)
+		if(ignite >= 3)
+			add_modifier(/datum/modifier/fire/stack_managed/intense, 60 SECONDS)

```

Comment:
If ignite = 2, you will have regular fire for 90s, but if ignite = 15, you'll only burn for 60s, intentional?

Reply:
Intentional, as 60s of intense fire is 600 damage, while 90 seconds of standard is 450 damage.